infrastructure/module/eks3/main.tf

```diff
@@ -18,6 +18,9 @@ module "eks" {
     coredns = {
       most_recent = true
     }
+    aws-ebs-csi-driver = {
+      most_recent = true
```

Comment:
Make sure you add **service_account_role_arn** whose value will come from IRSA module.